Core/source/core/pgp-hash.ts

```diff
@@ -2,17 +2,17 @@
 
 'use strict';
 
+import { util, Sha1, Sha256 } from 'openpgp';
 import { Buf } from './buf';
-import { openpgp } from './pgp';
 
 export class PgpHash {
 
   public static sha1UtfStr = async (string: string): Promise<string> => {
-    return openpgp.util.Uint8Array_to_hex(await openpgp.crypto.hash.digest(openpgp.enums.hash.sha1, Buf.fromUtfStr(string)));
+    return util.uint8ArrayToHex((new Sha1()).process(Buf.fromUtfStr(string)).finish().result);
   }
```

Comment:
is `sha1UtfStr` used anywhere in the library? I see `doubleSha1Upper`. Is that one used? If not, both can be removed.

Reply:
removed it